dex/testing/dcrdex/harness.sh

```diff
@@ -54,6 +60,15 @@ cat > "./markets.json" <<EOF
             "maxFeeRate": 100,
             "swapConf": 1,
             "configPath": "${TEST_ROOT}/btc/harness-ctl/alpha.conf"
+        },
+        "LTC_simnet": {
+            "bip44symbol": "ltc",
+            "network": "simnet",
+            "lotSize": 1000000,
+            "rateStep": 1000000,
+            "maxFeeRate": 20,
+            "swapConf": 2,
```

Comment:
Having this different than the other assets helped me catch the `fromAsset`/`toAsset` bug.

Reply:
A good catch.  Smart to keep it different.